content/guides/typescript.md

```diff
@@ -124,7 +137,7 @@ For more information see [this blog post](https://blogs.msdn.microsoft.com/types
 
 ## Importing Other Assets
 
-To use non code assets with TypeScript, we need to defer the type for these imports. This requires a `custom.d.ts` file which signifies custom definitions for TypeScript in our project. Let's set up a declaration for `.svg` files:
+To use non-code assets with TypeScript, we need to defer the type for these imports. This requires a `custom.d.ts` file which signifies custom definitions for TypeScript in our project. Let's set up a declaration for `.svg` files:
 
 __custom.d.ts__
```

Comment:
Can you please use a diff on the project to show where the file goes in the folder structure? Thanks!